models/src/main/java/com/foodie/FoodieBaseService.java

```diff
@@ -1,16 +1,21 @@
 package com.foodie;
 
+import java.lang.reflect.Field;
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Optional;
 
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageImpl;
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.domain.Sort;
 import org.springframework.data.util.Streamable;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 public interface FoodieBaseService<T extends FoodieBaseEntity, ID> {
```

Comment:
rename `ID`

Reply:
Renamed to Key

Reply:
Nah Key is better AS I am dealing with the primary key

Reply:
As long as it's all lowercase `key`